graveler/graveler.go

```diff
@@ -55,7 +56,7 @@ type Ref string
 // TagID represents a named tag pointing at a commit
 type TagID string
 
-// CommitParents commit's parents slice
+// CommitParents commit parents slice
```

Comment:
Not sure this comment adds information :-(

Reply:
Agree